src/build/env.py

```diff
@@ -97,6 +97,19 @@ def _restore_env(self):  # type: () -> None
     def _get_env_path(self, path):  # type: (str) -> Optional[str]
         return sysconfig.get_path(path, vars=self._env_vars)
 
+    def _symlink_file(self, path, new_subpath):  # type: (str, str) -> None
+        new_path = os.path.join(self.path, new_subpath)
+        dir_path = os.path.dirname(new_path)
+
+        if not os.path.exists(dir_path):
+            os.makedirs(dir_path)
+
+        if fs_supports_symlink():
+            os.symlink(path, new_path)
+        else:
+            import shutil
```

Comment:
this import should be at top levle, not in here IMHO.

Reply:
Sure. I just kept it here as it's last resort and won't be needed in the vast majority of the cases.

Reply:
Ops, turns out it was already up there :see_no_evil: